src/main/java/org/sonarsource/sonarlint/ls/SonarLintLanguageServer.java

```diff
@@ -163,13 +166,16 @@ static SonarLintLanguageServer bySocket(int port, Collection<URL> analyzers) thr
       // until https://github.com/microsoft/vscode-languageserver-node/pull/697 is released
       // params.getClientInfo().getName()
       String appName = (String) options.get("appName");
-      String ideVersion = appName + " " + params.getClientInfo().getVersion();
+      String workspaceName = (String) options.get("workspaceName");
```

Comment:
Is the workspace name stable during the entire LS life? What if you close a folder, or open a new one?

Reply:
In manual tests with a single folder, closing the open folder and opening a new one resulted in the LS shutting down and starting again with the new value. With an existing multi-folder workspace, the name remained stable. We need to test what happens with an "ad-hoc" multi folder workspace.